tests/manager/manager-ctor-auto_encryption-001.phpt

```diff
@@ -0,0 +1,83 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): auto encryption options
+--FILE--
```

Comment:
If `MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION` is undefined, `phongo_manager_set_auto_encryption_opts()` throws the following if it finds a `autoEncryption` key in a non-null `driverOptions`:

```c
phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "Cannot enable automatic field-level encryption. Please recompile with support for libmongocrypt using the with-mongodb-client-side-encryption configure switch.");
```

This test will fail when FLE is not enabled. I propose you create a `skip_if_not_libmongocrypt()` function based on `skip_if_not_libmongoc_crypto()` and `skip_if_not_libmongoc_ssl()` in `tests/utils/skipif.php`. It will be simpler than those, as you won't need logic to check for specific libs.

Reply:
Added the function, also added the counterpart and an additional test for when compiling without FLE as mentioned in the comment below.